src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Drawable getRandomFigure() {
+        return switch (random.nextInt(FIGURES_COUNT)) {
+            case 0 -> new Circle(10, colorSupplier.getRandomColor());
+            case 1 -> new Square(5, colorSupplier.getRandomColor());
+            case 2 -> new Rectangle(3, 7, colorSupplier.getRandomColor());
+            case 3 -> new RightTriangle(5, 9, colorSupplier.getRandomColor());
+            default -> new IsoscelesTrapezoid(4, 6, 5, colorSupplier.getRandomColor());
+        };
+    }
+
+    public Drawable getDefaultFigure() {
+        return new Circle(10, "WHITE");
```

Comment:
Avoid using magic numbers. The number 10 should be defined as a constant to give it a meaningful name and to avoid duplication.